File: SWEA/D2/1945. 간단한 소인수분해/간단한 소인수분해.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        int[] primes = {2, 3, 5, 7, 11};
        
        for (int t = 1; t <= T; t++) {
            int N = sc.nextInt();
            int[] counts = new int[5];
            
            for (int i = 0; i < 5; i++) {
                while (N % primes[i] == 0) {
                    counts[i]++;
                    N /= primes[i];
                }
            }
            
            System.out.print("#" + t + " ");
            for (int i = 0; i < 5; i++) {
                System.out.print(counts[i] + " ");
            }
            System.out.println();
        }
        sc.close();
    }
}

File: SWEA/D2/1945. 간단한 소인수분해/간단한 소인수분해.py
T = int(input())
primes = [2, 3, 5, 7, 11]

for t in range(1, T + 1):
    N = int(input())
    counts = [0] * 5
    
    for i in range(5):
        while N % primes[i] == 0:
            counts[i] += 1
            N //= primes[i]
            
    # 결과 출력 (언패킹 혹은 join 활용)
    print(f"#{t} {' '.join(map(str, counts))}")